MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,387 @@
+---
+layout: post
+title: Grouping in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Grouping editors in .NET MAUI DataForm (SfDataForm)
```

Comment:
Grouping the editors in .NET MAUI DataForm (SfDataForm)

Reply:
changed heading